src/components/ReanimatedSwipeable.tsx

```diff
@@ -317,10 +316,41 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       );
     };
 
+    const dispatchImmidiateEvents = useCallback(
+      (fromValue: number, toValue: number) => {
+        if (toValue > 0 && props.onSwipeableWillOpen) {
+          runOnJS(props.onSwipeableWillOpen)('left');
```

Comment:
```suggestion
         props.onSwipeableWillOpen('left');
```

This function is not a worklet so `runOnJS` isn't needed here.